src/main.js

```diff
@@ -1 +1,10 @@
-// Write your code here
\ No newline at end of file
+// Write your code here
+
+console.log(22+10);
+console.log(3*3);
```

Comment:
ასე ჯობს, კოდი უფრო წაკითხვადია
```suggestion
console.log(22 + 10);
console.log(3 * 3);
```